test/e2e/network_test.go

```diff
@@ -301,11 +301,13 @@ var _ = Describe("Podman network", func() {
 		session := podmanTest.Podman([]string{"network", "create", netName1})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(BeZero())
+		defer podmanTest.removeCNINetwork(netName1)
```

Comment:
Unfortunately this will blow up.
We cannot remove the network twice.
The test already removes them.

Reply:
@Luap99 good point. I've filed #7809 and will revert my attempt. Thanks.